beacon-chain/blockchain/process_block.go

```diff
@@ -620,9 +618,6 @@ func (s *Service) lateBlockTasks(ctx context.Context) {
 	if !s.inRegularSync() {
 		return
 	}
-	s.cfg.StateNotifier.StateFeed().Send(&feed.Event{
```

Comment:
don't we still need this? Missed slot feed.  https://github.com/prysmaticlabs/prysm/issues/12153 based on this it needs to send when it's late.

I know you mentioned 
>The other important change is that this event is now triggered by calls to FCU, rather than piggybacking onto missed slot and new head events. This means that payload attribute events are no longer a special case, leading to simpler code, and their emission for builders is in sync with where we would notify the engine API. 

but wouldn't it still need to be there?

Reply:
I need to check if this applies when the next proposer is not a tracked validator. Will try to ping metachris in the other issue for more insight.

Reply:
In the meantime I'll add it back in.

Reply:
https://github.com/prysmaticlabs/prysm/pull/14644/commits/4edea3ca411ffce609bc8638e893e28c75b636f6

Reply:
I believe it's still needed for untracked ( this was opened a while back, I updated to reflect the same as lighthouse https://github.com/prysmaticlabs/prysm/issues/14204) but it's related to the untracked and submitting it, the builder should only use the registered info though.

Reply:
This preserves the existing behavior: https://github.com/prysmaticlabs/prysm/blob/develop/beacon-chain/rpc/eth/events/events.go#L600-L604

If we want to change it I'd rather do that in a separate PR.

Reply:
> but it's related to the untracked and submitting it, the builder should only use the registered info though.

I can't quite make sense of your comment. Right now the code fires events for head updates and missed slots, whether or not the upcoming proposer is tracked. If it is tracked, the fee recipient for the tracked validator will be used. Is that the correct behavior?

Reply:
that's the correct behavior, just meant that regardless the builder should use the data from the validator registrations and not from the proposer attributes event